modules/zinit/service.go

```diff
@@ -31,11 +31,11 @@ const (
 
 // InitService represent a Zinit service file
 type InitService struct {
-	Exec    string
-	Oneshot bool
-	Test    string
-	After   []string
-	Log     LogType
+	Exec    string   `yaml:"exec,omitempty"`
+	Oneshot bool     `yaml:"oneshot,omitempty"`
+	Test    string   `yaml:"test,omitempty"`
+	After   []string `yaml:"after,omitempty"`
+	Log     LogType  `yaml:"log,omitempty"`
```

Comment:
I don't believe this relate to the PR but okay

Reply:
indeed, I already pushed it on master so I can remove it from here. https://github.com/threefoldtech/zosv2/commit/a36852a93f4e9b68db1be020c1602e19bb876df8